packages/target-decisioning-engine/src/index.spec.js

```diff
@@ -77,7 +78,7 @@ describe("TargetDecisioningEngine", () => {
     expect(typeof decisioning.getOffers).toBe("function");
   });
 
-  // eslint-disable-next-line jest/no-test-callback
+  // eslint-disable-next-line jest/no-done-callback
```

Comment:
hmmm, 🤔  we use done callbacks pretty often in tests.  better to disable this rule in the global config as an override so it is permitted.

Reply:
Done - turned it off globally in `eslintrc.js`